host_core/test/e2e/echo_test.exs

```diff
@@ -10,12 +10,12 @@ defmodule HostCore.E2E.EchoTest do
     ]
   end
 
-  @echo_key "MADQAFWOOOCZFDKYEYHC7AUQKDJTP32XUC5TDSMN4JLTDTU2WXBVPG4G"
-  @httpserver_link "default"
-  @httpserver_path "test/fixtures/providers/httpserver.par.gz"
+  @echo_key HostCoreTest.Constants.echo_key()
+  @httpserver_link HostCoreTest.Constants.default_link()
+  @httpserver_path HostCoreTest.Constants.httpserver_path()
 
   test "echo roundtrip", %{:evt_watcher => evt_watcher} do
-    {:ok, bytes} = File.read("test/fixtures/actors/echo_s.wasm")
+    {:ok, bytes} = File.read(HostCoreTest.Constants.echo_path())
```

Comment:
Same as previous comment, should this be `@echo_path` ?

Reply:
Yep, I'll refactor this